api/cinemaHall/CinemaHall.js

```diff
@@ -1 +1,18 @@
-//CinemaHallSchema
+const mongoose = require('mongoose');
+
+const CinemaHallSchema = new mongoose.Schema({
+  name: Number,
+  seats: [
+    {
+      type: Schema.Types.ObjectId,
+      ref: 'Seat',
+    },
+  ],
+  rows: Number,
```

Comment:
perhaps rows should have seats inside? something like {1: [seats], 2: [seats]} ?